packages/server/src/utilities/schema.ts

```diff
@@ -163,21 +168,31 @@ export function parse(rows: Rows, schema: TableSchema): Rows {
 }
 
 function isValidBBReference(
-  columnData: any,
-  columnSubtype: FieldSubtype.USER | FieldSubtype.USERS
+  data: any,
+  type: FieldType.BB_REFERENCE | FieldType.BB_REFERENCE_SINGLE,
+  subtype: FieldSubtype.USER | FieldSubtype.USERS
 ): boolean {
-  switch (columnSubtype) {
+  if (typeof data !== "string") {
+    return false
+  }
+
+  if (type === FieldType.BB_REFERENCE_SINGLE) {
+    if (!data) {
+      return true
+    }
```

Comment:
... is this correct?

Reply:
Yes, this was not considering required settings, as we don't do anywhere else. I just changed this specific use case to check it